challenge2/styles.css

```
@@ -0,0 +1,164 @@
+:root {
+    /* Colors */
+    --primary-color: #FCFCFC;
+    --complimentary-red: #EF4444;
+}
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+html {
+    font-size: 62.5%;
+    font-family: "Ubuntu", sans-serif;
+}
+
+header p {
+    font-size: 1.8rem;
+}
+
+.main-section {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-top: 20px;
+    padding-bottom: 25%;
+    background-color: var(--primary-color);
+    flex-direction: column;
+}
+
+.form-container {
+    width: 100%;
+    padding: 26px 8px 20px 20px;
+}
+
+h1 {
+    font-size: 3rem;
+    font-weight: 500;
+    padding-bottom: 3rem;
+}
+
+.title p {
+    font-family: "Manrope", sans-serif;
+    font-size: 1.6rem;
+    font-weight: 400;
+    padding-bottom: 10px;
+}
+
+.title a {
+    padding-bottom: 2rem;
+}
+
+.info {
+    display: flex;
+}
+
+form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+}
+
+.image-container img {
+    display: none;
+}
+
+.input-group {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+    padding-bottom: 20px;
+}
+
+.input-wrapper {
+    display: flex;
+    align-items: center;
+    width: 100%;
+    border-bottom: 1px solid #5A5F70;
+    position: relative; /* Add this to position the eye icon */
+}
+
+.link {
+    color: var(--complimentary-red);
+    text-decoration: underline;
+}
+
+input {
+    border: none;
+    outline: none;
+    flex-grow: 1; /* Allow input to take up available space */
+}
+
+.eyeImage {
```

To be consistent with other class names:

```suggestion
.eye-image {
```